velox/docs/functions/spark/array.rst

```diff
@@ -112,6 +112,26 @@ Array Functions
         SELECT array_sort(array(NULL, 1, NULL)); -- [1, NULL, NULL]
         SELECT array_sort(array(NULL, 2, 1)); -- [1, 2, NULL]
 
+.. spark:function:: array_sort(array(T), function(T,U)) -> array(T)
+    :noindex:
+
+    Returns the array sorted by values computed using specified lambda in ascending
+    order. U must be an orderable type. Null elements will be placed at the end of
```

Comment:
Would you clarify what kind of lambda is expected here?

nit: U -> \``U\`` to referring to the variable

Would you document the NaN behavior?

Also document the behavior if the comparator function returns null.